scripts/run.sh

```diff
@@ -72,15 +72,15 @@ elif [ $1 == "test" ]; then
         echo
 
         if [[ $backend_rc == 0 && $frontend_rc == 0 ]]; then
-            echo "All tests pass!!!"
-            if [[ $GITHUB_HEAD_REPO_URL == "https://github.com/mozilla/balrog.git" ]];
+            if [ '$GITHUB_HEAD_REPO_URL' == "https://github.com/mozilla/balrog.git" ];
```

Comment:
I'll work something out soon.
I'm surprised as the previous way worked on my PC locally. I'm trying to reproduce it again using some conditions.